src/lib.rs

```diff
@@ -1496,14 +1496,14 @@ pub use zerocopy_derive::TryFromBytes;
 ///
 /// [undefined behavior]: https://raphlinus.github.io/programming/rust/2018/08/17/undefined-behavior.html
 /// [github-repo]: https://github.com/google/zerocopy
-/// [`try_from_ref`]: TryFromBytes::try_from_ref
+/// [`try_ref_from`]: TryFromBytes::try_ref_from
 /// [*valid instance*]: #what-is-a-valid-instance
 #[cfg_attr(feature = "derive", doc = "[derive]: zerocopy_derive::TryFromBytes")]
 #[cfg_attr(
     not(feature = "derive"),
     doc = concat!("[derive]: https://docs.rs/zerocopy/", env!("CARGO_PKG_VERSION"), "/zerocopy/derive.TryFromBytes.html"),
 )]
-// TODO(#5): Update `try_from_ref` doc link once it exists
+// TODO(#5): Update `try_ref_from` doc link once it exists
```

Comment:
Is this TODO done? Can we remove it while we're here?

Reply:
Done!